index.html

```diff
@@ -1,276 +1,240 @@
+                  </li>
+                  <li class="media_li"><a href="#" target="_blank"><img src="./images/Medium-icon.svg" alt="Medium icon"/></a>
+                  </li>
+                </ul>
+              </div>
+              <div class="get-b">
+              <a href="#" class="btn btn-mastery ">Get my resume</a>
             </div>
-        </div>
-        <div>
-          <ul>
-            <li aria-expanded="true" aria-haspopup="true">
-              Framework
-             
-              <i class="fa fa-angle-right"></i>
-            </li>
-          </ul>
-           <hr>
-        </div>
-        <div>
-          <ul>
-            <li aria-expanded="true" aria-haspopup="true">
-              Skills
-             
-              <i class="fa fa-angle-right"></i>
-            </li>
-          </ul>
-          <hr>
-        </div>
-      </div>
-    
-
     
-  </section>
-    </div>
+            </div>  
+        <div class="aim">
+          
+          <img src="./images/Skill-List.png" alt="portfolio"/>
+        </div>   
+    </section>
     <footer class="contact_container">
       <div class="contact_bk">
-     <div class="container flex">
-       <h1 class="feedback">Contact me</h1>
-       <p class="msg">
-        if you have an application you are interested in developing,
-        a feature that you need built or a project that needs coding. 
-        I'd love to help with it
-       </p>
-       <div>
-         <form action="https://formspree.io/f/mvodnloy" method="post">
-           <input type="text" aria-required="true" maxlength="30" name="name" placeholder="Patrick Rejoice">
-           <input type="email" aria-required="true" maxlength="30" name="email" placeholder="kapojames@gmail.com">
-           <textarea aria-required="true" maxlength="500" name="message" placeholder="Write your message here"></textarea>
-           <button class="btn" type="submit">Get in Touch</button>
-         </form>
-       </div>
-       <div class="line"></div>
-     </div>
-    </div>
-    </footer>
-  </div> 
-</body>
+        <div class="container">
+          <h1 class="contact_header">Contact me</h1>
+          <p class="contact_p">
+            if you have an application you are interested in developing,
+            a feature that you need built or a project that needs coding. 
+            I'd love to help with it
+          </p>
+          <div>
+            <form action="https://formspree.io/f/mvodnloy" method="post">
+              <input type="text" aria-required="true" maxlength="30" name="name" placeholder="Patrick Ndimukulaga">
+              <input type="email" aria-required="true" maxlength="30" name="email" placeholder="kapojames@gmail.com">
+              <textarea aria-required="true" maxlength="500" name="message" placeholder="Write your message here"></textarea>
+              <button class="btn" type="submit" >Get in Touch</button>
+            </form>
+          </div>
```

Comment:
MOBILE VERSION:

- [ ] Kindly follow the same placeholder name as given in the Figma design.
- [ ] Kindly make sure to follow the given Figma design and layout the elements according to it.
- [ ] Kindly make sure to reduce the extra space on the right side of the contact form as per the Figma design.
- [ ] Make sure to correctly place the background image as per the Figma design.
- [ ] Make sure the button is to the left side and not the centre as per the Figma design.
- [ ] Kindly follow the design as much as possible (e.g., font, colours, images, tests, margins) as per the Figma design.
- [ ] Make sure to follow the correct colour for placeholder text as per the Figma design.

- Figma design:
<img width="236" alt="Screenshot 2022-05-05 at 1 41 40 AM" src="https://user-images.githubusercontent.com/70269507/166839774-b9b2b880-8dec-4a75-8005-6e2d1a02fa0d.png">

- Your design:
<img width="382" alt="Screenshot 2022-05-05 at 2 10 44 AM" src="https://user-images.githubusercontent.com/70269507/166839863-4188fe16-e55d-4565-abc3-9a6b7d1d1865.png">